podvm/Makefile.inc

```diff
@@ -7,9 +7,10 @@
 # It is to be included via `include ../../podvm/Makefile.inc` in
 # each of the cloud provider podvm image Makefiles
 
+PARENT_DIR := $(dir $(lastword $(MAKEFILE_LIST)))../
```

Comment:
I'm not sure on this, but would `PODVM_DIR` be a clearer name than `PARENT_DIR`?

Reply:
It is pointing to the parent of the podvm directory i.e. the cloud-api-adaptor root. I can use the ROOT_DIR here too if you think that is clearer?

Reply:
Thanks - that's clearer to me and the matches the others.